src/index.html

```diff
@@ -5,8 +5,460 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
     <link rel="stylesheet" href="./styles/main.scss">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a href="index.html" class="header__logo">
+        <img src="images/logo.png" alt="moyo logo">
+      </a>
+
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__list-item">
+            <a
+            href="index.html"
+            class="nav__link is-active"
+            >
+              apple
+            </a>
+          </li>
+
+          <li class="nav__list-item">
+            <a
+            href="index.html"
+            class="nav__link"
+            >
+              samsung
+            </a>
+          </li>
+
+          <li class="nav__list-item">
+            <a
+            href="index.html"
+            class="nav__link"
+            >
+              smartphones
+            </a>
```

Comment:
Still not fixed from the previous review, you need to fix it everywhere
```suggestion
            <a
              href="index.html"
              class="nav__link"
            >
              smartphones
            </a>
```